common/views/components/Buttons/Button/Button.js

```diff
@@ -45,7 +45,7 @@ const ButtonInner = ({
 type ButtonProps = {|
   ...ButtonInnerProps,
   tabIndex?: string,
-  url?: string,
+  externalLink?: string,
```

Comment:
❓

Reply:
We need to use the Button component with an external link (i.e. not a NextLink), for the authentification. This was what the url prop was already doing, but the name wasn't obvious.

I think the button component is probably doing too much, but this works for now.

Reply:
> I think the button component is probably doing too much

Why not use the chance to split it out?

Reply:
It's on my todo list (just not at the top)